server/test/write_authzmodel.go

```diff
@@ -15,6 +15,7 @@ import (
 	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
 )
 
+// TODO: update these to SchemaVersion1_1 models
```

Comment:
Can we just add an issue for this with a link to this line number and remove this comment here? That way we have some book keeping to come back around to. I'll forget about this TODO comment if it's not an issue :)